src/pages/ProductPage/ProdDetail/ProdDetail.jsx

```diff
@@ -0,0 +1,23 @@
+import "./ProdDetail.scss";
+
+export default function ProdDetail({
+  company,
+  model,
+  description,
+  originalPrice,
+  price,
+  discount,
+}) {
+  return (
+    <div className="prodDetail">
+      <h2>{company}</h2>
+      <h1>{model}</h1>

```

Comment:
`H1` to `H6` are ways to define the most important heading of your web page: Being `H1` the most important and `H6` less important.

For example, if you have a Blog Post, and you open up a post, the very important heading that you will have is the `Title` of your post. Because you want to inform everybody that it is the main content of this page. So doing it right you will transform the robot's life easier to find you and index you correctly on search pages like Google.

You will never use H1'H6 for font-size purposes, if you want that a text that's not too important on the page be bigger, you use a CSS class for that.

So if `model` is your most important heading here, this is correct.